mediapipe/calculators/ovms/openvinomodelserversessioncalculator.h

```diff
@@ -34,6 +34,7 @@
 namespace mediapipe {
 
 using ovms::OVMSInferenceAdapter;
+extern OVMS_LogLevel OVMS_LOG_LEVEL;
```

Comment:
Why do we introduce global variable?

Reply:
This is only easy way to get access to setting from inference calculator into session calculator.